tests/unittests/QuantizationTest.cpp

```diff
@@ -430,29 +448,50 @@ TEST(Quantization, enableRowwiseQuantizedFullyConnectedSymmetric) {
 
   TensorQuantizationParams inputTQP = chooseQuantizationParams(
       -1.0, 6.0, quantization::Schema::Symmetric, ElemKind::Int8QTy);
-  TensorQuantizationParams fcTQP = chooseQuantizationParams(
+  TensorQuantizationParams matmulTQP = chooseQuantizationParams(
+      0.0, 10.0, quantization::Schema::Symmetric, ElemKind::Int8QTy);
+  TensorQuantizationParams batchedaddTQP = chooseQuantizationParams(
       0.0, 10.0, quantization::Schema::Symmetric, ElemKind::Int8QTy);
   TensorQuantizationParams biasTQP = chooseQuantizationParams(
       0, 20, quantization::Schema::Symmetric, ElemKind::Int8QTy);
 
   EXPECT_EQ(inputTQP.offset, 0);
-  EXPECT_EQ(fcTQP.offset, 0);
+  EXPECT_EQ(matmulTQP.offset, 0);
+  EXPECT_EQ(batchedaddTQP.offset, 0);
   EXPECT_EQ(biasTQP.offset, 0);
 
+  LoweredInfoMap loweredMapForQuant;
+  ::glow::lower(F, &loweredMapForQuant, EE.getBackend());
+
+  // Get the MatMul node and the Bactched_Add node.
+  Node *MatMul, *BatchedAdd;
+  for (Node &N : F->getNodes()) {
+    if (N.getKind() == Kinded::Kind::MatMulNodeKind) {
+      MatMul = &N;
+    }
+    if (N.getKind() == Kinded::Kind::BatchedAddNodeKind) {
+      BatchedAdd = &N;
+    }
+  }
+
```

Comment:
You should probably `ASSERT_TRUE()` on the pointers here to make sure they're correctly found.